### ISSUE TITLE
no attribute 'ActionModule' when using ansible 2.6.20 

### ISSUE THREAD

**Issue by arrfab** [ansible-community/ara#98]
## What is your ARA installation like ?
Ara installed in virtualenv, with pip like ansible 2.6.20 (on CentOS 8)
```
pip list|egrep 'ara|ansible'
ansible                   2.6.20   
ara                       1.3.0    
```

## What is happening ?
When calling ansible-playbook with a working playbook, it starts with 
```
Operations to perform:
  Apply all migrations: admin, api, auth, contenttypes, db, sessions
Running migrations:
  No migrations to apply.

PLAY 
```
and then the tasks until it reaches to one template task and then (after the OK: part): 

```
ERROR! Unexpected Exception, this is probably a bug: module 'ansible.plugins.action.service' has no attribute 'ActionModule'
```
The playbook stops at this stage, while the tasks that ran are availble in ara
some -vvv output : 
```
ERROR! Unexpected Exception, this is probably a bug: module 'ansible.plugins.action.service' has no attribute 'ActionModule'
the full traceback was:

Traceback (most recent call last):
  File "/home/arrfab/virtualenvs/ansible26/bin/ansible-playbook", line 118, in <module>
    exit_code = cli.run()
  File "/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ansible/cli/playbook.py", line 122, in run
    results = pbex.run()
  File "/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ansible/executor/playbook_executor.py", line 156, in run
    result = self._tqm.run(play=play)
  File "/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ansible/executor/task_queue_manager.py", line 292, in run
    play_return = strategy.run(iterator, play_context)
  File "/home/arrfab/ansible/CentOS_dev/mitogen-0.2.8/ansible_mitogen/strategy.py", line 350, in run
    lambda: run(iterator, play_context)
  File "/home/arrfab/ansible/CentOS_dev/mitogen-0.2.8/mitogen/core.py", line 633, in _profile_hook
    return func(*args)
  File "/home/arrfab/ansible/CentOS_dev/mitogen-0.2.8/ansible_mitogen/strategy.py", line 350, in <lambda>
    lambda: run(iterator, play_context)
  File "/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ansible/plugins/strategy/linear.py", line 247, in run
    action = action_loader.get(task.action, class_only=True)
  File "/home/arrfab/ansible/CentOS_dev/mitogen-0.2.8/ansible_mitogen/strategy.py", line 135, in wrap_action_loader__get
    klass = action_loader__get(name, **get_kwargs)
  File "/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ansible/plugins/loader.py", line 381, in get
    obj = getattr(self._module_cache[path], self.class_name)
AttributeError: module 'ansible.plugins.action.service' has no attribute 'ActionModule'
[mux  20272] 17:43:59.021152 D mitogen: Waker(fd=11/13): disconnecting
```

Wondering if that conflicting with mitogen (version mitogen-0.2.8) that is used also with strategy_plugin = mitogen_linear

**Comment by dmsimard** [comment]
@arrfab thanks! I've admittedly not tried mitogen with ara recently so I can't say for sure whether or not it is related. Are you able to reproduce the issue without mitogen ?

**Comment by arrfab** [comment]
it seems that env variables are getting in the way somehow.
I initially tested with exported variables proposed by `python3 -m ara.setup.env` so in my case I had this : 
```
export ANSIBLE_CALLBACK_PLUGINS=${ANSIBLE_CALLBACK_PLUGINS:-}${ANSIBLE_CALLBACK_PLUGINS+:}/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ara/plugins/callback
export ANSIBLE_ACTION_PLUGINS=${ANSIBLE_ACTION_PLUGINS:-}${ANSIBLE_CALLBACK_PLUGINS+:}/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ara/plugins/action
```
Then I tried through another shell but same python virtualenv, but this time I modified ansible.cfg and added (as also suggested by `python3 -m ara.setup.ansible` : 
```
callback_plugins=/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ara/plugins/callback
action_plugins=/home/arrfab/virtualenvs/ansible26/lib/python3.6/site-packages/ara/plugins/action
```
And this time it worked just fine ... :-)

**Comment by dmsimard** [comment]
@arrfab oh! this was actually reported and fixed by @sherve-korem yesterday: https://github.com/ansible-community/ara/commit/58e849735490ad6f5677343885c3ce1e6a9304c4

**Comment by arrfab** [comment]
oh ! now that I look at this, the error is visible and wasn't even looking at the exported variables ! :-)
I guess we can close this one, but good thing is that we know that it works great with mitogen too : so it's fast on execution and also having now good reports :-)

**Comment by dmsimard** [comment]
@arrfab that's good to hear! The fix will land in 1.3.1.
